File: rest/usage-records/list-get-example-3/list-get-example-3.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'date'
require 'rubygems'          # This line not needed for ruby > 1.8
require 'twilio-ruby'

# Get your Account Sid and Auth Token from https://www.twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Loop over records and print out a property for each one
@client.account.usage.records.list(
  :category   => 'calls-inbound',
  :start_date => Date.new(2012, 9, 1),
  :end_date   => Date.new(2012, 9, 30)).each do |record|
    puts record.price
  end

File: rest/usage-records/list-get-example-3/list-get-example-3.5.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import TwilioRestClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = TwilioRestClient(account_sid, auth_token)

# A list of record objects with the properties described above
records = client.usage.records.list(
    category="calls-inbound", start_date="2012-09-01", end_date="2012-09-30"
)
